apps/survey/contracts/test/mocks/BadToken.sol

```diff
@@ -20,6 +20,6 @@ contract BadToken is MiniMeToken {
     }
 
     function totalSupplyAt(uint _blockNumber) public constant returns(uint) {
-        return 1;
+        return _blockNumber - _blockNumber + 1;
```

Comment:
This is way too hacky ;) You can disable the warning by making `_blockNumber` an unnamed parameter:

```
function totalSupplyAt(uint) public constant returns (uint)
```